connectors/bigtable/flink-connector-gcp-bigtable/src/main/java/com/google/flink/connector/gcp/bigtable/utils/ErrorMessages.java

```diff
@@ -29,4 +29,8 @@ public class ErrorMessages {
             "Unsupported type, use Bytes for more complex types: ";
     public static final String BASE_NO_NESTED_TYPE =
             "Nested Rows mode require all non-key fields to be of type ";
+    public static final String SERIALIZER_ERROR =
+            "Error while serializer element to RowMutationEntry: ";
+    public static final String METRICS_ENTRY_SERIALIZATION_WARNING =
+            "Error while serializing RowMutationEntry for metrics, entry will be counted as 0 bytes. This error doesn't affect the job. Error: ";
```

Comment:
should this be a warning instead of an error?

Reply:
ummm, it's technically an error, but we are logging it as WARNING since it actually doesn't affect the job